src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,26 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    className={classNames(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': todo.completed,
+      },
+    )}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    {todo.user && (
+      <UserInfo user={todo.user} />
+    )}
+  </article>
 );
```

Comment:
Also you can destructuring `todo`
```suggestion
export const TodoInfo: React.FC<Props> = ({ todo }) => {
	const {
		completed,
		title,
		user,
	} = todo;
	return (
	  <article
	    className={classNames(
	      'TodoInfo',
	      {
	        'TodoInfo--completed': completed,
	      },
	    )}
	  >
	    <h2 className="TodoInfo__title">{title}</h2>
	
	    {user && (
	      <UserInfo user={user} />
	    )}
	  </article>
	);
};
```